examples/maps_heatmaps/test/initializeDB.ts

```diff
@@ -0,0 +1,353 @@
+import { database } from "../src/index";
+import * as firebase from "firebase";
+import * as geofirestore from "geofirestore";
+
+/**
+ * initalize the database
+ * to run in the terminal-  npm run initialize-db
+ */
+
+const allCoordinates = [
```

Comment:
move coordinates to a text file?

Reply:
we moved coordinates to a separate text file